dataeng/resources/remote-config.sh

```diff
@@ -14,6 +14,11 @@ set -xe
 
 env | sort
 
+# Creating python3.7 virtual env
+PYTHON_VENV="python_venv"
+virtualenv --python=python3.7 --clear "${PYTHON_VENV}"
```

Comment:
I would suggest to update it to python3.8 and also test the job with you branch and see if anything breaks. It looks good to me but just make sure you have tested it.